src/i965_encoder_vp8.c

```diff
@@ -2087,6 +2087,19 @@ i965_encoder_vp8_get_picture_parameter(VADriverContextP ctx,
         vp8_context->ref_arf_frame = NULL;
     }
 
+    if (!is_intra) {
+        if (!vp8_context->ref_last_frame) {
+            if (vp8_context->ref_gf_frame)
+                vp8_context->ref_last_frame = vp8_context->ref_gf_frame;
+            else
+                vp8_context->ref_last_frame = vp8_context->ref_arf_frame;
```

Comment:
Is it possible last/gf/arf frames are all NULL frame? Maybe we can add an assert() for this case.

Reply:
For non-key frame, one of last/gf/arf must be non-NULL.

Reply:
This is normal case. Actually for normal case, last frame must be non-NULL for P frames.